netket/sampler/metropolis.py

```diff
@@ -596,3 +596,32 @@ def MetropolisGaussian(hilbert, sigma=1.0, **kwargs) -> MetropolisSampler:
 
     rule = GaussianRule(sigma)
     return MetropolisSampler(hilbert, rule, **kwargs)
+
+
+def MetropolisAdjustedLangevin(
+    hilbert, dt=0.001, chunk_size=None, **kwargs
+) -> MetropolisSampler:
+    """This sampler acts on all particle positions simultaneously
+    and takes a Langevin step [1].
+
```

Comment:
Can you expand this description with a few formulas and a more extensive explanation? Linking to wikipedia is great but most other samplers give a precise description of what they do..

Reply:
w should mention also that this is for continuous space models.